src/jsutils.h

```diff
@@ -57,21 +60,30 @@ size_t flash_strlen(const char *str);
 /// Copy a string from flash to RAM
 char *flash_strncpy(char *dest, const char *source, size_t cap);
 
+/// Compare a string in memory with a string in flash
+int flash_strcmp(const char *mem, const char *flash);
```

Comment:
I guess we could `#define flash_strcmp strcmp` when not on ESP8266 and it might help us remove a bunch of ifdefs from the code itself?